src/utils/wallets.ts

```diff
@@ -11,11 +11,20 @@ import {
 } from '@solana/wallet-adapter-wallets'
 import { WalletAdapterNetwork } from '@solana/wallet-adapter-base'
 import {
+  createDefaultAddressSelector,
   createDefaultAuthorizationResultCache,
+  createDefaultWalletNotFoundHandler,
   SolanaMobileWalletAdapter
 } from '@solana-mobile/wallet-adapter-mobile'
 
 export const getWalletAdapters = (network: WalletAdapterNetwork) => [
+  new SolanaMobileWalletAdapter({
+    appIdentity: { name: 'Goosefx App' },
+    authorizationResultCache: createDefaultAuthorizationResultCache(),
+    addressSelector: createDefaultAddressSelector(),
+    cluster: network,
+    onWalletNotFound: createDefaultWalletNotFoundHandler()
+  }),
```

Comment:
I moved this to the top of the list _because_ it gets filtered out:

* on desktop
* when on a non-supported mobile device (eg. iOS)

You can see this in the desktop test plan video.